client/components/addons/addon-collection/container.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react'
+import { observer, inject } from 'mobx-react'
+import PropTypes from 'prop-types'
+
+import AddonCollectionPage from './addon-collection'
+
+@inject('addon')
+@observer
+class AddonCollectionPageContainer extends Component {
+	static propTypes = {
+		addon: PropTypes.object.isRequired
+	}
+
+	render () {
+		const { addon } = this.props
+
+		return (
+			<AddonCollectionPage
+				addons={addon.addonsSortedByMostRecent}
```

Comment:
You possibly shouldn't be specifying the addons here as now if you want to reuse this component on another page you're tied to having them sorted by most recent. Perhaps pass them in as a prop, and let the pages handle that contextually?